### DISCUSSION TITLE
Shape Rotation?

### DISCUSSION THREAD

**Discussion by loyx2** [AlexeyBoiko/DgrmJS#27] [Ideas]
Great work Alexey! 
Is shape rotation available? This would be nice to have.

**Comment by AlexeyBoiko** [comment]
Hi, thanks!
Currently no. Can you provide example of diagramm where rotation is neede?

**Reply by loyx2** [reply]
A simple example would be an arrow shape (with a tail) where I could rotate to point to any direction; i.e. 90degrees, 45degrees, etc.

![image](https://user-images.githubusercontent.com/90814544/224505803-232d6b81-d778-4cff-8b59-3576130d38c2.png)

**Reply by AlexeyBoiko** [reply]
Thanks for suggestion. I add this idea to to-do list.

For lines I am planning to add point in the middle, with this point user can change line path. Arrow at the end will rotate.
![image](https://user-images.githubusercontent.com/29090032/224617086-bc48ec52-e6f7-432f-97a0-84def6ba4d15.png)

**Reply by loyx2** [reply]
That will be great Alexey - Thank you!  

If it is possible too to have an option of connecting shapes with non-curvy line as well :)

![image](https://user-images.githubusercontent.com/90814544/224640643-e214c51c-5d68-4ad8-877a-cb8f518085fd.png)
